bitnami/kafka/templates/svc-headless.yaml

```diff
@@ -8,15 +8,13 @@ spec:
   type: ClusterIP
   clusterIP: None
   ports:
-    - name: tcp-kafka
+    - name: tcp-client
```

Comment:
Why client? Wouldn't it be "service" in any case? Same for `svc.yaml`.

Reply:
Because it's the port for communications with clients, while the other port is the one for inter-broker communications.

Reply:
OK, it was just a bit confusing for me since in the end it is using the listener port. Wouldn't `tcp-listener` or something similar be easier to understand? In any case, feel free to keep it as-is.